pootle/apps/pootle_store/urls.py

```diff
@@ -19,9 +19,9 @@
     url(r'^xhr/stats/checks/?$',
         'get_qualitycheck_stats',
         name='pootle-xhr-stats-checks'),
-    url(r'^xhr/stats/overview/?$',
-        'get_overview_stats',
-        name='pootle-xhr-stats-overview'),
+    url(r'^xhr/stats/?$',
```

Comment:
Just wondering why we keep the `?`. These are XHR calls so we can enforce these URLs to always end on `/` on our JS code.